.github/workflows/nodejs.yml

```
@@ -0,0 +1,19 @@
+name: Node CI
+
+on: push
+jobs:
+  build:
+    runs-on: macos-latest
```

Let's standardize on ubuntu-latest for builds.
```suggestion
    runs-on: ubuntu-latest
```